### PR TITLE
Feat: privacy page

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
# Feat: privacy page

- Add a Privacy page which displays contacts (people) and apps
  - I decided to keep the Contacts page as it is today and create a new Privacy page which essentially displays the contacts as well. This is to make the the feature flag logic simpler, and to start migrating from contacts to audit list more gradually, but this page reuses a lot of the Contacts page components and functionality for now.
  - For now, the only app displayed is hardcoded in the AgentList (a single Thing created just for the purpose of having something to display)

- Add the privacy page to feature flag

- Make Tabs more generic so they can be reused throughout the app

- Add tests

Note: Prism version needs to be updated to a proper patch before merging

## To test:

- Verify that you can see the Privacy tab with a WebID included in the feature flag
- Verify that you cannot see the Privacy tab with a WebID not included in the feature flag
- Verify that Contacts page works as usual

# Checklist

- [x] All acceptance criteria are met.
- [ ] Relevant documentation, if any, has been written/updated.
- [ ] The changelog has been updated, if applicable.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by kevin-inrupt — APPROVED** [review]
Privacy page displayed appropriately for different users.

File: components/agentList/index.jsx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+import useProfiles from "../../src/hooks/useProfiles";
+import ProfileLink from "../profileLink";
+import SearchContext from "../../src/contexts/searchContext";
+import { deleteContact, vcardExtras } from "../../src/addressBook";
+import ContactsDrawer from "../contactsList/contactsDrawer";
+import ContactsEmptyState from "../contactsList/contactsEmptyState";
+
+const useStyles = makeStyles((theme) => createStyles(styles(theme)));
+
+// termporarily using mock data for apps for dev purposes until we have audit list
```
Comment (ajacksified) [inline components/agentList/index.jsx:61]:
```suggestion
// FIXME: temporarily use mock data for apps for dev purposes until we have audit list
```

"FIXME" makes searching the codebase for things we forgot easier.

File: components/pages/privacy/index.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    },
+    {
+      label: "People",
+      testid: TESTCAFE_ID_TAB_PEOPLE,
+      value: foaf.Person,
+    },
+    {
+      label: "Apps",
+      testid: TESTCAFE_ID_TAB_APPS,
+      value: "app", // this string for now until we define what type the app contact will be
```
Comment (ajacksified) [inline components/pages/privacy/index.jsx:55]: Same here and below at line 61:

```suggestion
      // FIXME: replace with app contact type iri
      value: "app", // this string for now until we define what type the app contact will be
```

You may also want to make "app" a constant to make it easy to refactor later.

File: components/pages/privacy/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import React from "react";
+import { fireEvent, render, waitFor } from "@testing-library/react";
```
Comment (ajacksified) [inline components/pages/privacy/index.test.jsx:23]: Some linter warnings below.

**Review by ajacksified — CHANGES_REQUESTED** [review]
Try to clean up as many of the linter warnings as possible; I know some are from commented tests, although I'd think it.skip would be good enough rather than commenting them out.

**Review by KyraAssaad — APPROVED** [review]
It works and it looks great!

**Review by ajacksified — APPROVED** [review]

**Pull request merged**